types/marked/index.d.ts

```diff
@@ -34,6 +35,15 @@ declare namespace marked {
      */
     function lexer(src: string, options?: MarkedOptions): TokensList;
 
+    /**
+     * @param src String of markdown source to be compiled
+     * @param links Array of links
+     * @param options Hash of options
+     * @return String of compiled HTML
+     */
+
+    function inlineLexer(src: string, links?: string[], options?: MarkedOptions): string;
```

Comment:
It looks like `links` should be required here

Reply:
Whoops, absolutely right. I'll update.

Reply:
Should be good to go now.